File: herzarika/InerySolution/Transfer_Token.mjs
import { Api, JsonRpc, JsSignatureProvider } from '../dist/index.js'

const url = process.env.NODE_URL

const json_rpc = new JsonRpc(url)
const private_key = process.env.PRIVATE_KEY;

const account = process.env.INERY_ACCOUNT
const actor = process.env.INERY_ACCOUNT
const token = process.env.TOKEN_TRANSFER
const signature  = new JsSignatureProvider([private_key]);

const api = new Api({
    rpc: json_rpc,
    signatureProvider: signature
})

async function transfer(from, to, quantity, memo){
    try{
        const tx = await api.transact({
            actions:[
                {
                  account,
                  name:"transfer",
                  authorization:[
                        {
                            actor,
                            permission:"active"
                        }
                    ],
                    data:{
                        from, to, quantity, memo
                    }
                }
            ]
        },{broadcast:true,sign:true})

        console.log(tx)
        console.log(tx.processed.action_traces[0].console)
    }catch(error){
        console.log(error)
    }
}

transfer(actor, "haruhime", token, "this for HRH for you bro")

File: miladachuya/transfertoken.py
import os
from api.cline_trans import Cline_trans
from api.keys_trans import INRKey
from ineryjs import Api, JsonRpc, RpcError, JsSignatureProvider

os.environ.update({k.lower(): v for k, v in os.environ.items()})

TOKEN_CONTRACT = os.getenv("TOKEN_CONTRACT")
TRANSFER_ACCOUNT = os.getenv("INERY_ACCOUNT")
TOKEN_SYMBOL = os.getenv("TOKEN_SYMBOL")
TRANSFER_QUANTITY = os.getenv("TRANSFER_QUANTITY")
TRANSFER_MEMO = os.getenv("TRANSFER_MEMO")

json_rpc = JsonRpc(os.getenv("INERY_NODE_URL"))
signature_provider = JsSignatureProvider([os.getenv("PRIVATE_KEY")])

api = Api({
    "rpc": json_rpc,
    "signature_provider": signature_provider
})

async def TRANSFER_token(token_contract, transfer, quantity, memo):
    try:
        tx = await api.transact({
            "actions": [
                {
                    "account": TOKEN_CONTRACT,
                    "name": "TRANSFER",
                    "authorization": [
                        {
                            "actor": TRANSFER_ACCOUNT,
                            "permission": "active"
                        }
                    ],
                    "data": {
                        "quantity": quantity,
                        "memo": memo
                    }
                }
            ]
        }, {"broadcast": True, "sign": True})

        print(tx["processed"])
        print(tx["processed"]["action_traces"][0]["act"])
    except RpcError as error:
        print(f"ERROR: Can't TRANSFER token symbol {TOKEN_SYMBOL}")
        print(f"DETAILS: {error.details[0]['message']}")

TRANSFER_token( TOKEN_CONTRACT , TRANSFER_ACCOUNT , TRANSFER_QUANTITY , TRANSFER_MEMO )